prov/shm/src/smr_ep.c

```diff
@@ -185,9 +188,11 @@ static int smr_progress_inline(struct smr_cmd *cmd, struct smr_ep_entry *entry)
 	if (ret != cmd->hdr.op.size) {
 		FI_WARN(&smr_prov, FI_LOG_EP_CTRL,
 			"recv truncated");
-		return -FI_EIO;
+		entry->resv = FI_EIO;
```

Comment:
If you're using the resv field to store a value then it's not really reserved.

Reply:
Ahh, in my head that was standing for "result value". I'll change the name.